### DISCUSSION TITLE
Web UI doesn't show packages after container restart

### DISCUSSION THREAD

**Discussion by damodaranr** [verdaccio/verdaccio#3173] [Q&A]
I have verdaccio running as a docker container on Azure Container Instances. It has a persistent storage account mounted for saving the packages, conf files and htpasswd files. If the container gets restarted the WebUI stops showing available packages despite successful login, but we are able to access the packages through command line after authenticating. This is automatically fixed if we publish a new version of a package. However, it needs to be done for every single available package in the repository for WebUI to start showing themg again.
Is there a fix or workaround for this? Running verdaccio 5.2.2

**Comment by juanpicado** [comment]
If seems the persist is not working correctly, same as https://github.com/verdaccio/verdaccio/discussions/3172

**Reply by damodaranr** [reply]
How to get the persist working correctly? Should there be something done to generate the verdaccion-db.json. The packages themselves are present in the storage but they don't show up in the UI until I publish a new version and then the history on the UI shows all the previous versions as well.

**Reply by damodaranr** [reply]
Also note, that the packages only do not show up in the main landing page after login, but using saved direct link to the packages, the Web UI is able to show the details.

**Reply by damodaranr** [reply]
For Instance, after a container restart, when signed in as seen on the chrome network tab, http://xyz.westus.azurecontainer.io:1571/-/verdaccio/data/packages  returns []

but the direct link
http://xyz.westus.azurecontainer.io:1571/-/web/detail/packagename works and produces a page to browse the contents of it.

**Reply by lazearrr** [reply]
Has there been any movement on this? I'm experiencing the exact same issue as @damodaranr , packages are persisting in the /storage/data folder in my azure storage account, htpasswd is in my /storage folder in the azure storage account, but the .verdaccio-db.json file isn't being updated, I tried manually adding the package names in .verdaccio-db.json but still not showing in the web UI

**Comment by abrito92** [comment]
Having the same issue here. I am not sure if this is permission issues within azure or what. Testing locally it works flawlessly, so my best guess is permission. 

That said, i don't really know where to start looking.

**Reply by lazearrr** [reply]
I had to switch to using a virtual machine with an attached disk image, looks like there was work being done on a Azure Storage plugin for Verdaccio but haven't seen much movement on that.

**Reply by abrito92** [reply]
Thanks for the sugestion. I guess i will go to that route to.  I've been trying to make this work for a while, but guess i have to really change the deploy.
